### DISCUSSION TITLE
Access logs from outside container?

### DISCUSSION THREAD

**Discussion by ps-tunnelsoft** [MaxWaldorf/guacamole#133] [Q&A] [answered]
Would it be possible to adjust the container in a way that I can access the guacamole logs from outside via `docker logs guacamole`.

I used different guacamole container before and got used to just running the docker cmd if something was not working. Now I have to access the file `/var/log/supervisor/guacd_error.log`

**Comment by MaxWaldorf** [comment, marked as answer]
Just mount the log file somewhere...
